great_expectations/expectations/expectation.py

```diff
@@ -329,7 +331,11 @@ class Expectation(metaclass=MetaExpectation):
         2. Data Docs rendering methods decorated with the @renderer decorator. See the
     """
 
-    version: ClassVar = ge_version
+    class Config:
+        arbitrary_types_allowed = True
+        extra = pydantic.Extra.allow
```

Comment:
Essentially `**kwargs` but necessary as an intermediate step before we add defined params throughout the hierarchy